static/app/views/settings/organizationMembers/organizationMemberRow.tsx

```diff
@@ -162,7 +159,7 @@ export default class OrganizationMemberRow extends PureComponent<Props, State> {
               {isInviteSuccessful && <span>{t('Sent!')}</span>}
               {!isInviting && !isInviteSuccessful && (
                 <Button
-                  disabled={!canAddMembers}
+                  disabled={!canAddMembers && !(canInvite && isInviteFromCurrentUser)}
```

Comment:
this is a bit hard to grok, would writing it as `!canAddMembers || !canInvite || !isInviteFromCurrentUser` still work?

Reply:
for me it's easier to think about when the button should be enabled, which is when either
- `canAddMembers` is true
- `canInvite` is true AND `isInviteFromCurrentUser` is true

but you're right this logic is hard to follow so I'll make a new boolean or something for readability 🫡